packages/app/src/components/SourceForm.tsx

```diff
@@ -631,6 +631,21 @@ export function TraceTableModelForm({ control, watch }: TableModelProps) {
           placeholder="SpanName"
         />
       </FormRow>
+      <FormRow
```

Comment:
This field was already being inferred for the default OTEL trace table, now we allow the user to modify it if needed.